packages/auth/src/providers/cognito/utils/signInHelpers.ts

```diff
@@ -630,3 +633,19 @@ export function isMFATypeEnabled(
 	if (!mfaTypes) return false;
 	return mfaTypes.includes(mfaType);
 }
+
+export async function isUserAuthenticated() {
```

Comment:
Nit pick but the current name makes it sound like this behaves like a predicate (i.e. returns a bool to know if user is signed in).

```suggestion
export async function assertUserNotAuthenticated() {
```

Reply:
Yeah agree. Sounds more like a predicate. I changed it already. Thanks